packages/language-server/test/completion-items-spec.ts

```diff
@@ -832,6 +832,109 @@ describe("the UI5 language assistant Code Completion Services", () => {
     expectLspKind("UI5UnknownKey", CompletionItemKind.Text);
   });
 
+  context("deprecated", () => {
+    const NO_DEPRECATED_SUGGESTIONS = {
+      codeAssist: { deprecated: false, experimental: true },
+    };
+    const ALLOW_DEPRECATED_SUGGESTIONS = {
+      codeAssist: { deprecated: true, experimental: true },
+    };
+
+    function testDeprecated(xmlSnippet: string, suggestionLabel: string): void {
```

Comment:
if there is only use usage than extracting the test logic to a function can make it harder to follow the flow of a test without reducing LOC.

In our case I think that `testDeprecated` and `testExperimental` can be **combined** to a single utility as they seem mostly duplicated except being passed different "settings" objects.

Reply:
Done